tfutils/base.py

```diff
@@ -144,29 +146,29 @@ def __init__(self,
         if not os.path.isdir(self.cache_dir):
             os.makedirs(self.cache_dir)
 
-        if restore:
-            self.restore_model(force_fetch=force_fetch)
+        self.load_model(force_fetch=force_fetch)
 
         self.start_time_step = time.time()  # start timer
 
-    def restore_model(self, force_fetch=False):
+    def load_model(self, force_fetch=False):
         """
         Fetches record then uses tf's saver.restore
         """
         # fetch record from database and get the filename info from record
-        if self.dosave:
+        if self._restore:
```

Comment:
model should be restored if it can be, and if _restore is true -- whether or not dosave (and further saving) will happen.